setup.json

```diff
@@ -21,14 +21,16 @@
             "pseudo = aiida_pseudo.data.pseudo.pseudo:PseudoPotentialData",
             "pseudo.psf = aiida_pseudo.data.pseudo.psf:PsfData",
             "pseudo.psml = aiida_pseudo.data.pseudo.psml:PsmlData",
-            "pseudo.upf = aiida_pseudo.data.pseudo.upf:UpfData"
+            "pseudo.upf = aiida_pseudo.data.pseudo.upf:UpfData",
+            "pseudo.psp8 = aiida_pseudo.data.pseudo.psp8:Psp8Data"
```

Comment:
please move this one line up to keep alphabetical ordering. Some for the family entry point